src/test/java/com/example/gradecalculator/entities/SchoolYearTest.java

```diff
@@ -11,72 +11,70 @@ public class SchoolYearTest {
 
     @Test
     public void testGetId() {
-        SchoolYear schoolYear = new SchoolYear("2022-2023", LocalDate.of(2022, 9, 1), LocalDate.of(2023, 6, 30), new GradeSystem(1L, "Grade System"));
+        SchoolYear schoolYear = new SchoolYear("2022-2023", LocalDate.of(2022, 9, 1), LocalDate.of(2023, 6, 30));
```

Comment:
you needed to change this variable everywhere -> just make it available to all trests in the beginning, so you do not have to declare it so often